docs/appendix/troubleshooting.md

```diff
@@ -3,227 +3,103 @@ title: "Troubleshooting"
+
+- **Description:** When following our [installation guides](/installation/), the very first `ssh` connection you make to your newly-imaged {{< glossary_tooltip term_id="board" text="board" >}} will likely encounter this message.
+   This is expected: `ssh` is advising you that it has not yet connected to this address, and prompts you for how to proceed.
+
+- **Solution:** The message will ask `Are you sure you want to continue connecting?`.
+   Type `yes` and then press the return key to continue with the connection.
+   You should receive a successful confirmation message similar to: `Warning: Permanently added 'hostname.local' to the list of known hosts.`
+   This is only required for the very first `ssh` connection you make to a newly-installed board.
+
+### ssh: connect to host hostname.local port 22: Host is down
+
+- **Description:** Your computer is not able to connect to the {{< glossary_tooltip term_id="board" text="board" >}} through `ssh`.
+
+- **Solution:** Ensure that both your computer and the board itself are connected to the internet, and verify each of the following:
+   - If you are on a Windows computer, be sure that you do not have an outoging firewall preventing `ssh` connections over port `22`.
+   - Your `ssh` connection string should resemble the following: `ssh username@hostname.local`. Be sure that you match hostname, username, and password exactly to what you initially configured when imaging your board.
+   - If you are still unable to connect, try power cycling the board and trying your connection again.
+   - If that fails, try re-imaging your board following the [installation guide](/installation/) appropriate for your board.
+   If using the Raspberry Pi [installation guide](/installation/prepare/rpi-setup/), be sure to carefully enter the configuration details under the **Advanced Options** (gear icon) button on the [Raspberry Pi imager](https://www.raspberrypi.com/software/) before you re-image your board.
+
+### ssh: connect to host hostname port 22: Connection timed out
+
+- **Description:** Your computer is not able to connect to the {{< glossary_tooltip term_id="board" text="board" >}} through `ssh` before reaching a timeout.
+
+- **Solution:** Depending on your local network, it may take a minute or two for your `ssh` connection to successfully reach the board.
+   - If you encounter this error once or twice, try your `ssh` command again and wait until it completes.
+   - If the error persists, try the solutions presented for the `Host is down` error message above.
+
+## Common Viam App Errors
+
+### Failed to connect; retrying
+
+- **Description:** The [Viam app](https://app.viam.com) is unable to communicate with your robot, and will attempt to reconnect every few seconds until it is able to do so.
+When a robot is disconnected, it will continue to run with its locally-cached current configuration, but is not accessible for control or configuration remotely through the Viam app.
```

Comment:
```suggestion
When a robot is disconnected, it will continue to run with its locally-cached current configuration, but is not accessible for remote control or configuration through the Viam app.
```
[opt]